alacritty/src/config/bindings.rs

```diff
@@ -1023,6 +1028,7 @@ mod tests {
         b2.mode = TermMode::ALT_SCREEN;
 
         assert!(b1.triggers_match(&b2));
+        assert!(b2.triggers_match(&b1));
```

Comment:
Done on purpose to make every test symmetric, so it won't get borken in the future.